resources/assets/js/app/collections/voteAICollection.js

```diff
@@ -5,8 +5,10 @@ module.exports = BaseCollection.extend({
     model: Model,
     url: '/voteitems',
     initialize: function (models, options) {
-        if (options.parentUrl) {
-            this.parentUrl = options.parentUrl;
-        }
+        if (options)
+            if (options.parentUrl) {
+                this.parentUrl = options.parentUrl;
```

Comment:
implement this logic in BaseCollection, we have a lot of same initializers accross app